### PR TITLE
Feature/filter news backend

### PR DESCRIPTION
**Pull request by HristinaGlavcheva**

### CODE REVIEW

File: CryptoWorld.News/CryptoWorld.News.Core/Enumerations/NewsSorting.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace CryptoWorld.News.Core.Enumerations
+{
+    public enum NewsSorting
+    {
+        Soonest = 1,
```
Comment (ZdravkoBorisov) [inline CryptoWorld.News/CryptoWorld.News.Core/Enumerations/NewsSorting.cs]: Please rename this to Latest

File: CryptoWorld.News/CryptoWorld.News.Core/Enumerations/NewsSorting.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+        MostPopular = 2
+    }
```
Comment (ZdravkoBorisov) [inline CryptoWorld.News/CryptoWorld.News.Core/Enumerations/NewsSorting.cs:9]: Please add more filters.  For example filter news by specific dates, such as past week, past month, or a custom date range. Provide options to filter news based on geographical regions or countries - region Europe, Asia, etc

**Review by ZdravkoBorisov — CHANGES_REQUESTED** [review]

File: CryptoWorld.News/CryptoWorld.News.Core/Enumerations/NewsSorting.cs (hunk 1 above)
Comment (HristinaGlavcheva) [reply CryptoWorld.News/CryptoWorld.News.Core/Enumerations/NewsSorting.cs]: Done

File: CryptoWorld.News/CryptoWorld.News.Core/Enumerations/NewsSorting.cs (hunk 2 above)
Comment (HristinaGlavcheva) [reply CryptoWorld.News/CryptoWorld.News.Core/Enumerations/NewsSorting.cs:9]: Done

**Review by VelinArbov — APPROVED** [review]

**Review by ZdravkoBorisov — APPROVED** [review]

**Pull request merged**